app/templates/main/userProfile.html

```diff
@@ -257,5 +254,51 @@ <h6 align="left" class="d-inline-flex" id="unbanVolunteerEndDate">End Date: &nbs
     </div>
   </form>
 </div>
-{% endblock %}
 <!-- End Ban or Unban Modal -->
+{% for type in backgroundTypes %}
+    <div class="modal fade" id="historyModal-{{type.id}}" tabindex="-1" aria-hidden="true">
+      <div class="modal-dialog">
+        <div class="modal-content">
+          <div class="modal-header">
+            <h5 class="modal-title">Background Check History for </h5>
+            <button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="Close"></button>
+          </div>
+          <div class="modal-body">
+              <table class="table">
+                 <thead>
+                   <tr>
+                     <th scope="col">Status</th>
+                     <th scope="col">Date</th>
+                   </tr>
+                 </thead>
+                 <tbody>
+                   {% for entry in allEntries[type.id] %}
+                   <tr>
+                     <td>
+                        {% if loop.index == 1%}
+                            <h6>{{entry[0]}}</h6>
+                        {% else %}
+                            {{entry[0]}}
+                        {% endif %}
+
+                     </td>
+                     <td>
+                         {% if loop.index == 1%}
+                            <h6>{{entry[1]}}</h6>
+                         {% else %}
+                            {{entry[1]}}
+                         {% endif %}
```

Comment:
This display doesn't work. Also, use a class to add the bold text to the first item so you don't have to have a conditional in jinja